README.md

```diff
@@ -19,6 +19,11 @@ In short, change `url.bs` and submit your patch, with a
 [good commit message](https://github.com/whatwg/meta/blob/master/COMMITTING.md). Consider
 reading through the [WHATWG FAQ](https://whatwg.org/faq) if you are new here.
 
+If your patch makes normative (behavioural) changes, you will be asked to add or update
+[tests](https://github.com/web-platform-tests/wpt), and update the
+[jsdom/whatwg-url](https://github.com/jsdom/whatwg-url) implementation (see "Tests and
```

Comment:
I didn't use the word "official".

This came about from a private discussion I had with @annevk and @domenic, where they pointed me at jsdom/whatwg-url and said new changes to the standard would require people to update jsdom/whatwg-url.

I'm not sure of the "official" status of jsdom/whatwg-url, but it seems it is at least the de facto reference implementation. It's currently kept up-to-date (either by whoever changes the standard, or by someone else shortly afterwards) so that it passes all the tests, and I think it's good to keep it that way. It's best to tell people potentially changing the standard well in advance that this is an expectation, so that a) they'll change it themselves, and b) they are able to use it to test their changes and see how they affect the test suite.

(To put it another way, how can you reasonably expect someone to update WPTs without being able to run their changes against the entire test suite?)